ash/src/device.rs

```diff
@@ -582,7 +854,7 @@ impl Device {
             count_buffer_offset,
             max_draw_count,
             stride,
-        );
```

Comment:
I don't think it's good style to omit trailing semicolons in bodies of functions that return `()`, since propagating the value of the inner expression isn't intended or useful, and it's a footgun for future refactoring efforts that might add further statements below.